exercises/Python_Principles.ipynb

```diff
@@ -215,7 +183,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "type({ 'England': 'London', 'Scotland': 'Edinburgh', 'France': 'Paris'})"
```

Comment:
could we add a bit more information on dictionaries vs sets vs tuples? perhaps a brief description on why you might want to use one vs another?

Reply:
(unless we want to save this for later in the notebooks, for the case of simplicity)

Reply:
UPDATE: I see that these are covered in more detail in notebook 2, and that there is a comment to the effect that "we'll cover things more later" - so we can either leave as it is, or make that comment more explicitly refer to this content

Reply:
I do think, though, that in the relevant section of notebook 2 - on tuples, sets, etc - that we should touch on _why_ you may use them, as I don't think this is covered yet

Reply:
Yep agree. I did have a section on this on a first rough draft. I'll add this back in just now.